pages/spicedb/concepts/datastores.mdx

```diff
@@ -30,6 +30,44 @@ spicedb migrate head \
 
 For more information, refer to the [Datastore Migration documentation](/spicedb/concepts/datastore-migrations).
 
+## Read replicas for Postgres and MySQL
+
+SpiceDB can reference read-only replicas of permissions data stored in Postgres and MySQL that respect consistency requirements by detecting if a given revision has been replicated from the
+primary datastore before reading from the replica.
+If the requested revision is not found in the replica, the request will be redirected to the primary datastore.
+
+Strict read mode (available for Postgres only) does not check replicas for requested revisions before reading from them; instead, a revision check is inserted into the SQL for each read.
+This mode is useful for deploying replicas behind a load balancer than can transparently handle these requests.
+To reduce overhead of using read replicas, enable caching on CheckRevision calls to avoid checking the requested revision if the last checked revision is at least as fresh as the one
+requested.
+
+Includes a test to return whether the replicated request successfully falls back to the primary datastore.
+
+### Usage Notes
+
+- Maximum replica count is 16.
+- Note: be very careful when using this function.
+It is not safe to be used without knowledge of underlying datastore and replicas.
+Replicas will be checked for the requested revision before reading from them, which means that the read pool for the replicas must point to a stable instance of the datastore (not a load
+balancer).
+That means each replica node in the database must be configured as its own replica to SpiceDB, which each URI given distinctly.
+- Calls using strict read mode must reference replicas with strict mode enabled to ensure the query will fail with a revision unavailable error if the specified revision is not available.
```

Comment:
There is no "strict mode" in the database side. It's something we came up with, and I don't think we need to expose this implementation detail.

These bits read oddly, here is my proposal

```suggestion
- Do not use a load-balancer between SpiceDB and MySQL replicas, as SpiceDB cannot maintain consistency guarantees. You may use a load-balancer between SpiceDB and Postgres replicas. Listing individual hostnames is safe for both MySQL and Postgres replicas.
```

Reply:
After conferring, @jzelinskie and I reorganized the content a bit to incorporate your feedback by separating out mysql and postgres into separate subsections; this might help in the future too if other differences arise.